build.zig

```diff
@@ -83,16 +92,24 @@ const ExampleOptions = struct {
     optimize: OptimizeMode,
     lib_sokol: *CompileStep,
     mod_sokol: *Module,
+    package: ?*Build.Dependency = null,
+
+    fn packagePath(self: ExampleOptions, b: *Build) ?[]const u8 {
+        if (self.package) |dep|
+            return dep.path("").getPath(b);
+        return null;
+    }
```

Comment:
I have left the name generic. This is in case it is reused in another package. It's not only private to emsdk.
I should also change the local emsdk_path in the build_wasm function.